presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -16,7 +16,7 @@ function canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-'teradatalabs/cdh5-hive:4'}
```

Comment:
See the comment above :) We don't have the hdp-master-and-slave-in-one image (the dh5-hive equivalent) yet, but once we have it, no changes are needed in presto itself - just setting the env variable in `presto-checks` or wherever we run